frontend/src/views/web/challenge/my-submission.html

```diff
@@ -92,7 +92,7 @@
                                             <td ng-if="key.publication_url == ''"> None </td>
 
                                             <td class="val-style capitalize" ng-class="key.status">{{key.status}}</td>
-                                            <td class="align-center">{{key.execution_time}}</td>
+                                            <td class="align-center" ng-show="key.execution_time > 0">{{key.execution_time}}</td>
```

Comment:
Please add a default value `None` which will be displayed when the execution time is < 0.

Reply:
Done.

Reply:
Done.